grakn-graph/src/main/java/ai/grakn/graph/internal/TypeImpl.java

```diff
@@ -150,6 +150,7 @@ V addInstance(Schema.BaseType instanceBaseType, BiFunction<Vertex, T, V> produce
         }
 
         Vertex instanceVertex = getGraknGraph().addVertex(instanceBaseType);
+        getGraknGraph().getConceptLog().addedInstance(getLabel());
```

Comment:
Ideally yes but where would you put it? In the constructor? The constructor has no way of knowing if a vertex is new or old. Same for the factory. You could maybe put it in the `type` setter (because that is only called for new concepts) but that sounds odd.